spec/unit/recipes/source_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'spec_helper'
+
+describe 'prometheus::source' do
+  let(:chef_run) do
+    ChefSpec::SoloRunner.new(file_cache_path: '/var/chef/cache') do |node|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.